hypervisor/arch/x86/guest/vcpu.c

```diff
@@ -680,23 +694,26 @@ void schedule_vcpu(struct acrn_vcpu *vcpu)
 static uint64_t build_stack_frame(struct acrn_vcpu *vcpu)
 {
 	uint64_t rsp = (uint64_t)&vcpu->stack[CONFIG_STACK_SIZE - 1];
-	uint64_t *sp;
+	struct stack_frame *sp;
```

Comment:
rsp means the register of stack here? seems not too. It mean the stack top?
sp is a little bit confusing. It means stack frame pointer, right?  how about sfp? or frame or sf?